lang/cs/Org.Apache.REEF.Utilities/Logging/Logger.cs

```diff
@@ -100,11 +116,6 @@ public static List<TraceListener> TraceListeners
             }
         }
 
-        public static void SetCustomLevel(Level customLevel)
```

Comment:
I've thought about it some more, and I feel it will save us some trouble with deprecation if we keep name `CustomLevel` for global level (and keep this method for setting it), and use `InstanceLevel` for the newly introduced field. If any external code uses `CustomLevel` field, this way they won't have to rename it.